<launch>
   <node name="system_node" pkg="ros_homebot" type="system_node.py" output="screen" clear_params="true">
      <!--<param name="verbose" type="bool" value="1" />-->
   </node>
</launch>
